bin/problem.py

```diff
@@ -260,11 +260,13 @@ def validators(problem, validator_type, check_constraints=False):
                  glob(problem.path / (validator_dir + '_format_validators'), '*'))
 
         if len(paths) == 0:
-            if validator_type != 'output_format':
-                warn(f'No {validator_type} validators found.')
-            problem._validators[key] = False
-            return False
-        if validator_type == 'output_format' and problem.interactive and len(paths) > 1:
+            if validator_type == 'output_format':
+                log(f'No {validator_type} validators found (not required).')
```

Comment:
not needed to add this, since it will be clear to the user this is a LOG and not WARN or ERROR.
```suggestion
                log(f'No {validator_type} validators found.')
```